src/facility/facility.service.ts

```diff
@@ -115,6 +121,10 @@ export class FacilityService {
             const facility = await this.facilityRepository.findOne({id});
             if(facility)
             {
+                for(const i in facility.photos)
+                {
+                    facility.photos[i] = "https://"+process.env.CDN_URL+"/"+facility.photos[i];
```

Comment:
can you change this to `https://${process.env.CDN_URL}/facility.photos[i]`;

Reply:
\` https://${process.env.CDN_URL}/facility.photos[i];`/

Reply:
fixed